src/openms/source/FORMAT/MzTab.cpp

```diff
@@ -3027,10 +3053,20 @@ Not sure how to handle these:
         if (!is_fixed)
         {
           String unimod = res_mod.getUniModAccession();
+
```

Comment:
this whole block looks awfully like copy and pasted a couple of times. Can you make that a function?

Reply:
:)

Reply:
when I am through with the other issues :P